src/org/loklak/harvester/YoutubeScraper.java

```diff
@@ -205,7 +206,8 @@ public static JSONObject parseVideo(final BufferedReader br) throws IOException
     }
     
     private static long parseNumber(String n) throws NumberFormatException {
-        return Long.parseLong(numberfix.matcher(n).replaceAll(""));
+        String withoutSpaces = CharMatcher.WHITESPACE.removeFrom(n);
```

Comment:
YoutubeScraper fails with NumberFormatException on number which contains an whitespaces. Long.parseLong expect that you give it number without whitespaces. Current regexp does not remove whitespaces from string and I am decided remove spaces by using CharMatcher from Guava